### DISCUSSION TITLE
Daily 26/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#168] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Dificultades o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve con Angular, viendo videos y apuntes. Por la noche estuve en la clase de Inglés y vi la transmisión de la clase del Ispc.
Hoy trataré de empezar a trabajar en las tareas del proyecto. 
Dificultades: me lleva mucho tiempo entender y aprender para poder aplicarlo al proyecto.

**Comment by rpgrca** [comment]
Ayer estuve en la clase de web, también empecé con Angular, armé el encabezado y el pie de página, acomodé la página principal y armé la página de contacto. También estuve agregando pruebas unitarias al back-end, terminé de pulir las clases que estaban armadas. Hoy tengo el sincrónico de nivelación de inglés. Tal vez pueda armar otra página de Angular y me gustaría practicar armando un servicio para enviar los datos del formulario de contacto al back-end imaginario (que luego se hará en Django) para recordar algo de cómo era.

**Comment by BraianTroncoso** [comment]
Ayer realicé toda la página siguiendo psicología de color como también aplicado en el logo. La interfaz del usuario la hice mas amigable y acomodé un poco. De todas maneras fue un ejemplo porque tengo un par de ideas para mejorar en animación y otras cuestiones.
Tomé referencias de heladerías ya conocidas para crear la web siguiendo el mapa de sitio que tenemos como ejemplo.
Hoy planeo meterle un poco mas pero no tanto porque tengo clase de Ingles.

**Comment by Brian-Almada** [comment]
Ayer estuve viendo un poco más de Angular. Hoy voy a intentar poner en práctica algunas cosas.
